js/pihole/index.js

```diff
@@ -13,24 +13,6 @@ $(document).ready(function() {
 
     updateTopLists();
```

Comment:
I doubt that the data would be delivered faster than the charts are generated, but nevertheless this seems a good idea.